client/state/data-layer/wpcom/sites/atomic/software/index.js

```diff
@@ -7,14 +6,15 @@ import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import {
 	setAtomicSoftwareStatus,
 	setAtomicSoftwareError,
+	cleanAtomicSoftwareStatus,
 } from 'calypso/state/atomic/software/actions';
 import { registerHandlers } from 'calypso/state/data-layer/handler-registry';
 import { http } from 'calypso/state/data-layer/wpcom-http/actions';
 import { noRetry } from 'calypso/state/data-layer/wpcom-http/pipeline/retry-on-failure/policies';
 import { dispatchRequest } from 'calypso/state/data-layer/wpcom-http/utils';
-import { errorNotice } from 'calypso/state/notices/actions';
 
-const installSoftware = ( action ) =>
+const installSoftware = ( action ) => [
+	cleanAtomicSoftwareStatus( action.siteId, action.softwareSet ),
```

Comment:
we'd like to clean up the status to clean the UI in case it's a reattempt of installing.

Reply:
Sounds good, should we add that as a comment?

Reply:
done [here](https://github.com/Automattic/wp-calypso/pull/59394/commits/532edaa22bbd13e422c5b24a0a85de2a018d01fd)